src/buckets/components/csvUploader/CsvUploaderError.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import {SparkleSpinner} from '@influxdata/clockface'
+import {RemoteDataState} from 'src/types'
+
+const CsvUploaderError = () => (
+  <div className="line-protocol--spinner">
+    <p
+      data-testid="line-protocol--status"
+      className="line-protocol--status error"
+    >
+      Failed to Write Data
+    </p>
+    <SparkleSpinner loading={RemoteDataState.Error} sizePixels={220} />
+    <p className="line-protocol--status error">
+      Make sure your CSV is{' '}
```

Comment:
Is there a better way to add a space around the text in the `<a/>` tag?

Reply:
The only other options would be to add a `&nbsp;` instead, or set specific padding onto the `a` tag